### PR TITLE
Upgrade snowplow_utils package version (Close #43)

### LINKED ISSUE #43: Upgrade snowplow_utils package version
### Describe the bug
This package uses an older version of snowplow_utils which will cause dependancy conflicts if using other snowplow packages which depend on different versions of snowplow_utils in your dbt project.

### Steps to reproduce
1. Use both the latest mobile and web packages in your dbt project `packages.yml` file:
```yaml 
packages:
  - package: snowplow/snowplow_web
    version: 0.9.2
  - package: snowplow/snowplow_mobile
    version: 0.5.3
```

2. Run `dbt deps`.

### Expected results
No error, all packages install successfully.

### Actual results
```
Version error for package snowplow/snowplow_utils: Could not find a satisfactory version from options: ['>=0.12.0', '<0.13.0', '>=0.11.0', '<0.12.0']
```

### Additional context
Issue first raised in [discourse](https://discourse.snowplow.io/t/snowplow-utils-0-12-1-dbt-package-released/7499/2).

### PR DESCRIPTION
**Pull request by georgewoodhead**
## Description & motivation
We need to use the same version of snowplow_utils across all snowplow dbt packages to avoid version conflicts when installing multiple packages (e.g. web and mobile) in the same dbt project.

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [x] I have added tests & descriptions to my models (and macros if applicable)

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 packages:
   - package: snowplow/snowplow_utils
-    version: [">=0.11.0", "<0.12.0"]
+    version: [">=0.12.0", "<0.13.0"]
```
Comment (paulboocock) [inline packages.yml:3]: could this be `version: [">=0.11.0", "<0.13.0"]` instead? I figure the changes between 0.12 and 0.11 are non-breaking so both would work?
I suggest this as it sort of feels like the change as is should bump this to 0.6.0, whereas if we can just widen it (so both 0.11 and 0.12 work) then 0.5.4 is fine.

Comment (georgewoodhead) [reply packages.yml:3]: makes sense 👍

Comment (georgewoodhead) [reply packages.yml]: have amended commit with this change

**Pull request merged**